imports/api/responses.js

```diff
@@ -27,7 +27,8 @@ const responsePattern = {
   answer: Helpers.AnswerItem,
   answerWysiwyg: Match.Maybe(String),
   correct: Match.Maybe(Boolean), // whether or not this response was correct (used in a quiz with multiple attempts)
-  createdAt: Date
+  createdAt: Date,
+  feedback: Match.Maybe(String)
```

Comment:
I'm thinking that the feedback should live in the marks array inside of a grade, so that feedback is part of a grade and a response is only ever a thing that a student submits in the course of responding to a question. 
If the concern is to handle the case where a student submitted multiple responses to a question, then we could have an array of feedback for the different responses for that mark, but I think that is properly overkill